diff -uNr mysql-gui-tools-5.0r6/mysql-workbench/source/lua/WorkbenchImport.lua mysql-gui-tools-5.0r6.new/mysql-workbench/source/lua/WorkbenchImport.lua
--- mysql-gui-tools-5.0r6/mysql-workbench/source/lua/WorkbenchImport.lua	2006-03-04 00:23:11.000000000 +0100
+++ mysql-gui-tools-5.0r6.new/mysql-workbench/source/lua/WorkbenchImport.lua	2006-12-03 00:24:59.000000000 +0100
@@ -55,7 +55,7 @@
 function importDbd4(args)
   local filename= grtV.toLua(args[1])
   local p
-  local xF= 1.5
+  local xF= 3/2
   local yF= 2
   
   local callbacks= {
diff -uNr mysql-gui-tools-5.0r6/mysql-workbench/source/lua/Workbench.lua mysql-gui-tools-5.0r6.new/mysql-workbench/source/lua/Workbench.lua
--- mysql-gui-tools-5.0r6/mysql-workbench/source/lua/Workbench.lua	2006-07-03 23:40:15.000000000 +0200
+++ mysql-gui-tools-5.0r6.new/mysql-workbench/source/lua/Workbench.lua	2006-12-03 00:23:42.000000000 +0100
@@ -1683,10 +1683,10 @@
   local x= grtV.toLua(args[1])
   local y= grtV.toLua(args[2])
   local curView= grtV.getGlobal("/workbench/model/currentView")
-  local lowestX= 9999999999.0
-  local lowestY= 9999999999.0
-  local highestX= 0.0
-  local highestY= 0.0
+  local lowestX= 9999999999
+  local lowestY= 9999999999
+  local highestX= 0
+  local highestY= 0
 
   local i, c
 
