diff -Nur mysql-gui-tools-5.0r4.orig/mysql-gui-common/tools/grtsh/Makefile.am mysql-gui-tools-5.0r4.chng/mysql-gui-common/tools/grtsh/Makefile.am
--- mysql-gui-tools-5.0r4.orig/mysql-gui-common/tools/grtsh/Makefile.am	2006-09-06 00:04:50.000000000 +0200
+++ mysql-gui-tools-5.0r4.chng/mysql-gui-common/tools/grtsh/Makefile.am	2006-10-12 19:10:57.000000000 +0200
@@ -9,7 +9,7 @@
 if BUILD_NEWT
 newt_libs= $(top_builddir)/library/generic-runtime/source/.libs/libgrttextui.a\
 	$(top_builddir)/library/generic-runtime/newt/libmnewt.a \
-	@SLANG_LIB@ -ltermcap -lncurses
+	@SLANG_LIB@ -lncurses
 newt_flag= -DENABLE_TEXTUI
 endif
 
@@ -18,7 +18,7 @@
 endif
 
 if USE_READLINE
-rl_libs=-lncurses -lreadline -lhistory -ltermcap
+rl_libs=-lncurses -lreadline -lhistory
 grtsh_CPPFLAGS=-DUSE_READLINE 
 endif
 
